portfolio/src/main/java/com/google/sps/data/LoginStatus.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+/** 
+ * The object used to update client with user's log-in status and
+ * information to display on client side.
+ */
+public class LoginStatus {
+
+  private final Boolean isUserLoggedIn;
```

Comment:
nit: use primitive boolean unless you need functionality from the wrapper class.